lib/ex_prosemirror/modifier_helper.ex

```diff
@@ -1,7 +1,10 @@
-defmodule ExProsemirror.SchemaHelper do
+defmodule ExProsemirror.ModifierHelper do
   @moduledoc ~S"""
   Ecto schema helper that defines multiple macros to use inside an Ecto.Schema.
 
+  In contrast of `ExProsemirror.Schema`, this module should not be importer to your business schema.
+  It MUST be use to create custom block / marks.
```

Comment:
```suggestion
  It **MUST** be used to create custom block / marks.
```

the double `*` should make it bold, but check just to be sure, not sure the docs will read it like this.

[Reference](https://www.markdownguide.org/basic-syntax#bold)